docs/section22/Rule22-7.md

```diff
@@ -0,0 +1,40 @@
+
+# CHW&CW - Rule 22-7  
+
+**Rule ID:** 22-7  
+**Rule Description:** Baseline chilled water system that does not use purchased chilled water shall be modeled as primary/secondary systems.  
```

Comment:
the rule description should be updated to remove 'does not use purchased chilled water' and instead say ' all systems using chilled water..'